### PR TITLE
[MIG] [11.0] Migrate product multi category

### PR DESCRIPTION
**Pull request by damcar**
Sorry but I rebuilt the branch following the guide: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0
Pull request #294 is closed…
I reduced commits from « OCA Transbot … »
It’s ok ?
I sent CLA signed by email.

Thanks.

### CODE REVIEW

File: product_multi_category/README.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+=========================
+Product - Many Categories
+=========================
+
+This module Extends the existing functionality of Odoo Products
+(One product - One Catgory) to One product -> Many Categories
+
+*Note: This module was built generically but in focus of the Magento
+Odoo connector
+**Note: The additional categories are only for classification and does
```
Comment (LoisRForgeFlow) [inline product_multi_category/README.rst]: Please, close emphasis in notes: `**Note**`. Runbot is red because of it.

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
Minor comment inline

File: product_multi_category/README.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+not affect other operations
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/135/10.0
```
Comment (atchuthan) [inline product_multi_category/README.rst]: 11.0

**Review by atchuthan — CHANGES_REQUESTED** [review]
minor changes

**Review by atchuthan — APPROVED** [review]
Code Review & Runbot tested :+1: 
Minor changes only

**Review by pedrobaeza — APPROVED** [review]
I have fixed remaining points. Merging when CIs finish

**Pull request merged**